src/types/model/contentRelationship.ts

```diff
@@ -19,5 +19,6 @@ export interface CustomTypeModelContentRelationshipField<
 		customtypes?: readonly CustomTypeIDs[]
 		tags?: readonly Tags[]
 		allowText?: boolean
+		repeat?: boolean
```

Comment:
❓ Are we allowing content relationships to be repeatable? I thought we weren't, but correct me if I'm wrong.